src/gluonts/nursery/MSDA/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+__pycache__/
+dataset_utils/__pycache__/
+dataset_utils/__init__.py
+dump/
```

Comment:
Seems that this is just for a synthetic dataset. Do we really need to dump that somewhere?